lstchain/reco/r0_to_dl1.py

```diff
@@ -449,8 +450,12 @@ def r0_to_dl1(
 
                 if not is_simu:
                     dl1_container.ucts_time = 0
-                    # Assumes R1 event source filled the "dragon" timestamp:
-                    dl1_container.dragon_time = event.trigger.time
+                    # Assumes R1 event source filled the "dragon" timestamp
+                    # as MJD; we convert back to unix timestamp:
```

Comment:
Comment is wrong and code should be simpler.

Just do `dl1_container.dragon_time = event.trigger.time.unix`

Reply:
Is the scale still treated properly?

Reply:
unix is UTC, unix_tai is TAI, but wrong without fixing it (like in ctapipe_io_lst) or using the not-yet releases astropy 4.2.1

Reply:
Ok, will test it